app/controllers/concerns/importable.rb

```diff
@@ -0,0 +1,28 @@
+module Importable
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :current_organization, :current_user
+  end
+
+  def import_csv
+    if params[:file].present?
+      data = File.read(params[:file].path, encoding: "BOM|UTF-8")
+      resource_model.import_csv(data, current_organization.id)
+      flash[:notice] = "#{resource_model_name_plural} were imported successfully!"
+    else
+      flash[:error] = "No file was attached!"
+    end
+    redirect_back(fallback_location: { action: :index, organization_id: current_organization })
+  end
+
+  private
+
+  def resource_model
+    controller_name.classify.constantize
+  end
+
+  def resource_model_name_plural
+    controller_name.humanize
```

Comment:
Should this be `resource_model.name.underscore.humanize.pluralize`?

Reply:
@stevendaniels Initially I was doing that, then I realized that `controller_name` already came in snake_case and pluralized. 

Guess I can improve readability here by renaming the method `resource_model_humanized`.